docs/examples/hosting/docker/caddy/Caddyfile

```diff
@@ -0,0 +1,28 @@
+localhost {
+    encode gzip zstd
+
+    handle_path /server* {
+        reverse_proxy * server:3000 {
+            header_up Host {host}
+            header_up X-Real-IP {remote}
+            # header_up X-Forwarded-For {remote}
+            header_up X-Forwarded-Host {upstream_hostport}
+        }
+    }
+
+    handle {
```

Comment:
A separate `handle` block is required in order to have the SPA-routing not interfere with the proxy routing.